api/server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+
+const server = express();
+
+server.use(express.json());
+
+let id = 0;
+let state =[];
+
+server.get('/', (req, res) => {
+    res
+    .status(200)
+    .json('the line is hot');
+});
+
+server.get('/api/games', (req, res, next) => {
+    if (state.length === 0) {
+        res
+        .status(204)
+        .json([]);
```

Comment:
This isn't necessary since the empty array will be correctly returned in the else statement below. But I like that you are making sure to follow the README.